include/aca_dhcp_server.h

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef ACA_DHCP_SERVER_H
+#define ACA_DHCP_SERVER_H
+
+#include "aca_dhcp_programming_if.h"
+#include <map>
+
+using namespace aca_dhcp_programming_if;
+
+// dhcp server implementation class
+namespace aca_dhcp_server
+{
+struct dhcp_entry_data{
+	string ipv4_address;
+	string ipv6_address;
+	string port_host_name;
+};
+
+#define DHCP_ENTRY_DATA_SET(pData, pCfg)					\
+	do{														\
+		(pData)->ipv4_address = (pCfg)->ip_address;			\
```

Comment:
please use spaces for indentation instead of using tabs. The ACA project uses .clang-format placed in the root to automatically apply the code formatting. Please point your IDE (e.g. visual studio code or IDE of your choice) to leverage .clang-format.